src/java/com/test/TestQuiz.java

```diff
@@ -37,7 +37,15 @@ public void testCreateQuiz() throws Exception{
          assertEquals("Create fail.", true, a.createQuiz(quest, answer,id));
         
     }
-
+    
```

Comment:
Create test for method updateQuiz when:
- qid is not exist in the database
- question, answer null